### PR TITLE
Add direction and message counters,

### PR DESCRIPTION
**Pull request by joaobrittoneto**

### CODE REVIEW

File: src/DriverTypes.hpp [hunk 1]
```diff
@@ -274,7 +274,7 @@ struct AcousticConnection
  */
 struct DeviceSettings
 {
-    base::Time time;
+//    base::Time time;
```
Comment (doudou) [inline src/DriverTypes.hpp]: If you want to get rid of this line, _get rid of it_. Don't commit commented-out code.

File: src/UsblParser.cpp [hunk 2]
```diff
@@ -381,7 +408,7 @@ DeliveryStatus UsblParser::parseDeliveryStatus (string const &buffer)
 DeviceSettings UsblParser::parseCurrentSettings (string const &buffer)
 {
     DeviceSettings settings;
-    settings.time = base::Time::now();
+//    settings.time = base::Time::now();
```
Comment (doudou) [inline src/UsblParser.cpp]: Same comment ...

**Comment by joaobrittoneto** [comment]
@doudou,
What your opinion about the source level names?

**Comment by doudou** [comment]
Doesn't look bad. Maybe having an IN_AIR alias for MINIMAL just to make it clear ? (Just an idea)

**Comment by joaobrittoneto** [comment]
Done

**Pull request merged**